app/services/borrow_direct_reshare_client.rb

```diff
@@ -25,7 +27,10 @@ def get(path, **)
   end
 
   def get_json(path, **)
-    parse(get(path, **))
+    parse(check_response(get(path, **)))
+  rescue BorrowDirectError => e
```

Comment:
Is the client the right place to suppress this error, or should it be rescued at a higher level? (like, say, https://github.com/sul-dlss/mylibrary/blob/6d7795224927cea06d506a776bcce823bb0998e9/app/models/borrow_direct_reshare_requests.rb#L20?)

I guess I'm worried rescuing the errors this low in the stack could make it harder to debug problems.

Reply:
Makes sense. I moved the rescue up to `BorrowDirectReshareRequests#reshare_requests`.